src/config/constants.ts

```diff
@@ -20,6 +21,8 @@ export const NODE_ENV =
 
 export const SENTRY_DSN = process.env.REACT_APP_SENTRY_DSN;
 
+export const APP_VERSION = process.env.REACT_APP_VERSION || 'latest';
```

Comment:
Could we inject the latest git sha also here?

Reply:
Does it make sens to load it inside the app ? Anyway, we pass the SHA when deploying to dev in the Github actions workflow. For the default I think we can stay safe and use 'latest'.